CONTRIBUTING.md

```diff
@@ -58,20 +58,20 @@ Once you have babel running in watch mode, you can start making changes to parse
 
 If your pull request introduces a change that may affect the storage or retrieval of objects, you may want to make sure it plays nice with Postgres.
 
-* Run the tests against the postgres database with `PARSE_SERVER_TEST_DB=postgres npm test`. You'll need to have postgres running on your machine and setup [appropriately](https://github.com/parse-community/parse-server/blob/master/.travis.yml#L37) or use [`Docker`](#run-a-parse-postgres-with-docker).
+* Run the tests against the postgres database with `PARSE_SERVER_TEST_DB=postgres npm test`. You'll need to have postgres running on your machine and your configuratoin file setup [appropriately](https://github.com/postgres/postgres/blob/master/src/backend/utils/misc/postgresql.conf.sample#L63) along with [PostGIS](https://postgis.net) 2.2.0 higher installed, or use [`Docker`](#run-a-parse-postgres-with-docker).
```

Comment:
Since the build script has been moved from travis you should link to it.

https://github.com/parse-community/parse-server/blob/master/scripts/before_script_postgres.sh#L9

Reply:
I added to the section which shows how to run docker